gems/pending/VolumeManager/MiqLvm.rb

```diff
@@ -295,6 +302,436 @@ def parseString
   end
 end # class Lvm2MdParser
 
+# move to BinaryStruct
+def bs_offset(bs, attr)
```

Comment:
I don't think think this method should be created at the top-level...that seems bad.  If you can't get it into binary_struct gem for some reason, then at a minimum it should be scoped to some Util module.